### PR TITLE
qemu.tests.multi_vms_nic:no need ping itself ip

### PR DESCRIPTION
**Pull request by weliao**
1.Test no need ping itself ip.
2.When ping itself ip with -I eth0, will ping failed.

ID: 1375446

Signed-off-by: Wei Liao weliao@redhat.com

### CODE REVIEW

**Comment by CongLi** [comment]
Hi weliao,

Please update your commit log,
like qemu.tests.multi_vms_nic 

And could you add the operation of this PR to your summary?
The current is a little confused to me.

**Comment by weliao** [comment]
@CongLi  just add a judgment when src ip = dst ip , no do ping operation, because ping itself ip add -I eth0 will ping failed, new iputils packet set.

**Comment by huangyum** [comment]
(1/1) smp_16.4096m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.multi_vms_nic: PASS (2499.44 s)

LGTM.
Acked by: Yumei Huang yuhuang@redhat.com

**Comment by weliao** [comment]
@CongLi Could you help to review this PR again? thanks

File: qemu/tests/multi_vms_nics.py [hunk 1]
```diff
@@ -199,8 +199,9 @@ def file_transfer(session, src, dst):
...
             txt = "Ping test between %s and %s" % (src_ip_info[1], dst_ip[1])
             error.context(txt, logging.info)
-            ping(src_ip_info[2], src_ip_info[0], dst_ip[1], strict_check,
-                 flood_minutes)
+            if src_ip_info[1] != dst_ip[1]:
+                ping(src_ip_info[2], src_ip_info[0], dst_ip[1], strict_check,
+                     flood_minutes)
             txt = "File transfer test between %s " % src_ip_info[1]
             txt += "and %s" % dst_ip[1]
             error.context(txt, logging.info)
```
Comment (pingl2016) [inline qemu/tests/multi_vms_nics.py]: Shall we need to copy file on the same host? What is the point of the test?

**Comment by weliao** [comment]
@pingl2016  updated,  could you help to check again? thanks

**Comment by pingl2016** [comment]
Pls change you commit author by below steps:
    git config --global user.name "Your Name"
    git config --global user.email you@example.com
    git commit --amend --reset-author

**Comment by pingl2016** [comment]
Signed-off-by is missed in your commit log

**Comment by pingl2016** [comment]
Acked-by: Ping Li <pingl@redhat.com>

**Comment by xutian** [comment]
LGTM

**Pull request merged**